pages/test_analytics/ci_environments.md.erb

```diff
@@ -0,0 +1,55 @@
+# CI Environments
+
+Buildkite Test Analytics collectors automatically detect common continuous integration (CI) environments.
+If available, test collectors gather information about your test runs, such as branch names and build IDs.
```

Comment:
There's an open product/style question: "collectors" or "test collectors"? Both have been suggested in Basecamp discussions.

Reply:
Let's stick with the first for now, as other wise we'll have "Test Analytics Test Collectors".

Reply:
+1 to @plaindocs. We've been trying to align on 'collectors' (but have likely not eradicated other words). In calls with customers, folks seem to be able to understand what we mean with that as well when its in the context of Test Analytics already.

Reply:
LOLz so this convo progressed today to where.....we've renamed the actual collector packages to `buildkite-test-collectors` and we're in the process of renaming repos to things like `buildkite/test-collector-javascript` 🙃 .

Whether or not we're able to, or want to, carry those words through all our docs before launch -- I leave in your capable hands. (ie, Test Analytics Test Collectors still sounds silly, maybe it's not necessary to use the full name when used in certain contexts -- but I haven't tested this in docs to have a useful opinion.)

Reply:
I saw that update. I still think we can stick with `collectors` in the wording when the package is `test-collector-javascript`. At least for now.

Reply:
Agree